### ISSUE TITLE
Time not selected/visible if using inline mode

### ISSUE THREAD

**Issue by dleffler** [xdan/datetimepicker#31]
(v2.1.5) When initially displaying the date/time picker, the time widget will not scroll and the current time is not selected.  Clicking on the 'Today' button (or using firebug to 'unhide' the input field and then clicking in it) will enable the timepicker and select the time.

**Comment by jackdh** [comment]
Can anybody confirm a fix for this? I'm using the DTP within a model and finding that although the default time is selected it is not scrolling to that time.

**Comment by jackdh** [comment]
Encase anyone else needs a hack and slash method you can call this when loading the picker.

Adjust -351 to which ether suits your needs. 
```
setTimeout(function () {
    $('.xdsoft_time_variant').css('margin-top', '-351');
    }, 100)
```

**Comment by dleffler** [comment]
As a note, that last 'hack/slash' simply scrolls the time list to a set point (-351) instead of to what time is actually selected.

**Comment by jackdh** [comment]
Yes but I've found that the `defaultTime` does select the time but not scroll. SO if you have the default time to to 07:00 for example and then use that ugly hack it does make it look like it's scrolled to that time. As seen in the picture, I just clicked to show the picker. 
![jgptxvg 1](https://cloud.githubusercontent.com/assets/1907451/9976976/e2a751e0-5eee-11e5-9d0a-1bb64ec9550e.png)

**Comment by rnyberg** [comment]
Calling

  jQuery('.xdsoft_datetimepicker').trigger('afterOpen.xdsoft'); 

in my function that fades in the inline datetimepicker fixes this issue for me.

**Comment by dleffler** [comment]
The best fix for me with v2.5.3 is to add an additional line to the 'reset' command of the main plugin file
```
datetimepicker.trigger('xchange.xdsoft').trigger('afterOpen.xdsoft');
```
Then in the script which instantiates the widget within the $(document).ready(function(), add the following since the widget may or may not initially be visible
```
function waitForElementToDisplay(selector, time) {
    if($(selector).is(':visible')) {
        $('#idname').datetimepicker('reset');
    } else {
        setTimeout(function() {
            waitForElementToDisplay(selector, time);
        }, time);
    }
};
waitForElementToDisplay('#idname .xdsoft_datetimepicker', 1000);

**Comment by BennyRe** [comment]
@jackdh do you mean you've got the datetimepicker within a modal? If yes this comment is for future people searching for 'modal' in the issues. I had to spend another hour digging in the code till I found out it's because of the inline.

**Comment by Lodlaiden** [comment]
This is still an issue with an inline dateTimePicker in 2.5.4
I was able to get this to work based upon the above comments without editing any source.

```
setTimeout(
                function () {                    
                    $('#myDatePicker').datetimepicker('reset');
                }, 100);
            setTimeout(
                function () {
                    jQuery('.xdsoft_datetimepicker').trigger('afterOpen.xdsoft');
                }, 500);
```
